topics/assembly/tutorials/vgp_genome_assembly/tutorial.md

```diff
@@ -364,111 +410,185 @@ Once we have finished the genome profiling stage, we can start the genome assemb
 
-
 <div class = "hic" markdown="1">
 
+<!---- BEGINNING OF HiC SECTION --->
+<!---- BEGINNING OF HiC SECTION --->
+<!---- BEGINNING OF HiC SECTION --->
+<!---- BEGINNING OF HiC SECTION --->
+<!---- BEGINNING OF HiC SECTION --->
+
 ## HiC-phased assembly with **hifiasm**
 
 If you have the {Hi-C} data for the individual you are assembling with {HiFi} reads, then you can use that information to phase the {contigs}.
 
 > <hands-on-title>Hi-C-phased assembly with <b>hifiasm</b></hands-on-title>
-> 1. {% tool [Hifiasm](toolshed.g2.bx.psu.edu/repos/bgruening/hifiasm/hifiasm/0.18.8+galaxy1) %} with the following parameters:
->    - *"Assembly mode"*: `Standard`
->        - {% icon param-file %} *"Input reads"*: `HiFi_collection (trim)` (output of **Cutadapt** {% icon tool %})
->       - *"Hi-C R1 reads"*: `Hi-C_dataset_F`
->       - *"Hi-C R2 reads"*: `Hi-C_dataset_R`
+>**Step 1**: Run {% tool [Hifiasm](toolshed.g2.bx.psu.edu/repos/bgruening/hifiasm/hifiasm/0.19.8+galaxy0) %} with the following parameters:
+> 1. *"Assembly mode"*: `Standard`
+> 2. {% icon param-file %} *"Input reads"*: `HiFi_collection (trim)` (output of **Cutadapt** {% icon tool %})
+> 3. In *"Options for Hi-C-partition*" select `Specify`
+>     - *"Hi-C R1 reads"*: `Hi-C_dataset_F`
+>     - *"Hi-C R2 reads"*: `Hi-C_dataset_R`
 >
-> 2. After the tool has finished running, rename its outputs as follows:
->   - Rename the `Hi-C hap1 balanced contig graph` as `Hap1 contigs graph` and add a `#hap1` tag
->   - Rename the `Hi-C hap2 balanced contig graph` as `Hap2 contigs graph` and  add a `#hap2` tag
+>**Step 2**:. After the tool has finished running, rename its outputs as follows:
+>  1.  Rename the `Hi-C hap1 balanced contig graph` as `Hap1 contigs graph` and add a `#hap1` tag
+>  2.  Rename the `Hi-C hap2 balanced contig graph` as `Hap2 contigs graph` and  add a `#hap2` tag
 >
 {: .hands_on}
 
-We have obtained the fully phased contig graphs (as {GFA} files) of hap1 and hap2, but these must be converted to FASTA format for subsequent steps. We will use a tool developed from the VGP: **gfastats**. gfastats is a tool suite that allows for manipulation and evaluation of FASTA and GFA files, but in this instance we will use it to convert our GFAs to FASTA files. Later on we will use it to generate standard summary statistics for our assemblies.
+We have obtained the fully phased contig graphs (as {GFA} files) of hap1 and hap2, but these must be converted to FASTA format for subsequent steps. We will use a tool developed from the VGP: [`gfastats`](https://github.com/vgl-hub/gfastats). `gfastats` is a tool suite that allows for manipulation and evaluation of FASTA and GFA files, but in this instance we will use it to convert our GFAs to FASTA files. Later on we will use it to generate standard summary statistics for our assemblies.
+
+<div id="hap1_contigs">
```

Comment:
all hands on boxes can be linked to, see them on the page, they have a "link to here" button. https://training.galaxyproject.org/training-material/topics/assembly/tutorials/vgp_genome_assembly/tutorial.html#hands-on-convert-gfa-to-fasta

Reply:
One problem with anchors is that if the tutorial uses branching (`cyoa`) and the anchor within a branch that is not selected at the moment - it does  not work. Also, there may be hands-on boxes with identical names and see to be resolved with numbers that keep incrementing. Anyway - I did remove these.